mysql-plugin/src/test/java/io/cdap/plugin/mysql/MysqlPluginTestBase.java

```diff
@@ -208,18 +214,21 @@ private static void populateData(PreparedStatement ...stmts) throws SQLException
         pStmt.setDate(15, new Date(CURRENT_TS));
         pStmt.setTime(16, new Time(CURRENT_TS));
         pStmt.setTimestamp(17, new Timestamp(CURRENT_TS));
-        pStmt.setShort(18, (short) YEAR);
-        pStmt.setString(19, name);
+        pStmt.setTimestamp(18, new Timestamp(CURRENT_TS));
+        pStmt.setShort(19, (short) YEAR);
         pStmt.setString(20, name);
         pStmt.setString(21, name);
         pStmt.setString(22, name);
-        pStmt.setString(23, "char" + i);
-        pStmt.setBytes(24, name.getBytes(Charsets.UTF_8));
+        pStmt.setString(23, name);
+        pStmt.setString(24, "char" + i);
         pStmt.setBytes(25, name.getBytes(Charsets.UTF_8));
-        pStmt.setBlob(26, new SerialBlob(name.getBytes(Charsets.UTF_8)));
+        pStmt.setBytes(26, name.getBytes(Charsets.UTF_8));
         pStmt.setBlob(27, new SerialBlob(name.getBytes(Charsets.UTF_8)));
         pStmt.setBlob(28, new SerialBlob(name.getBytes(Charsets.UTF_8)));
         pStmt.setBlob(29, new SerialBlob(name.getBytes(Charsets.UTF_8)));
+        pStmt.setBlob(30, new SerialBlob(name.getBytes(Charsets.UTF_8)));
+        pStmt.setString(31, "Second");
+        pStmt.setString(32, "a,b");
```

Comment:
what do we check here?

Reply:
Here we set test values for `ENUM_COL` and `SET_COL` columns for Source test: https://github.com/vladhlinskiy/database-plugins/pull/8/files#diff-7a12ff7140fda3bf9bfe7eeef803de43R148